docker-entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+# This resolves the issue with a volume mount to `/conjur` overriding the `Gemfile.lock`
+# from `docker build`.
+cp "/tmp/Gemfile.lock" "Gemfile.lock"
```

Comment:
Seems a bit hacky but maybe the comment should just be expanded a bit more to specify more exactly that it's for running the container as-is with the mounted volume overlaid?

Reply:
done